text/0000-build-std/4-stage-1a.md

```diff
@@ -0,0 +1,1003 @@
+# Proposal
+[proposal]: #proposal
+
+Cargo configuration will contain a new key `build-std` under the `[build]`
+section ([?][rationale-build-std-in-config]), permitting one of two values -
+"off" ([?][rationale-build-std-off]) or "always", defaulting to "off":
+
+```toml
+[build]
+build-std = "always" # or `off`
+```
```

Comment:
Naming bikeshed: we've tended to use `"never"` as a complement to `"always"`, or use `true`/`false`